src/services/webhook.service.ts

```diff
@@ -6,6 +6,18 @@ import { Message } from "../model/message";
 import { Wallet } from "../model/wallet";
 import { Transaction } from "../model/transaction";
 
+export async function handleTransactionCreatedOrUpdated(
```

Comment:
this is now being used from both webhook and polling services, so maybe extract out of webhook service?